api-client/src/maps/index.ts

```diff
@@ -29,6 +29,7 @@ export type GeoJsonPoiProperties = {
   address?: string
   distance?: string
   thumbnail?: string
+  closeToOtherPoi?: boolean
```

Comment:
maybe i overseen it but is there any reason to make it optional? I think its `false` by default

Reply:
Good point